tests/testcontainers-env/src/main/java/org/alfresco/tas/AlfrescoStackInitializer.java

```diff
@@ -437,10 +478,27 @@ public String getPostgreSQLImage()
             return "postgres:" + envProperties.apply("POSTGRES_TAG");
         }
 
+        @Override
+        public String getMySQLImage()
+        {
+            return "mysql:" + envProperties.apply("MYSQL_TAG");
+        }
+
+        @Override
+        public DatabaseType getDatabaseType() {
+            String databaseTypeProperty = mavenProperties.apply("database.type");
```

Comment:
property name should be in static variable too imo

Reply:
I wanted to stay consistent with the class.